frontend/src/metabase/components/SchedulePicker.jsx

```diff
@@ -9,10 +9,17 @@ import { capitalize } from "metabase/lib/formatting";
 import { t } from "ttag";
 import _ from "underscore";
 
-export const HOUR_OPTIONS = _.times(12, n => ({
-  name: (n === 0 ? 12 : n) + ":00",
-  value: n,
-}));
+export const HOUR_OPTIONS = [
+  { name: "1:00", value: 1 },
+  { name: "2:00", value: 2 },
+  { name: "3:00", value: 3 },
+  { name: "4:00", value: 4 },
+  { name: "5:00", value: 5 },
+  { name: "6:00", value: 6 },
+  { name: "7:00", value: 7 },
+  { name: "11:00", value: 11 },
```

Comment:
This is also disabling 8-10 pm.

A 24-hour clock notation would be easier, and also less ambiguous :)

Reply:
True, but I think it's fine. Users still have options with other hours.